addons/website/static/src/snippets/s_popup/000.js

```diff
@@ -39,21 +45,27 @@ const PopupWidget = publicWidget.Widget.extend({
      * @private
      */
     _bindPopup: function () {
-        const $main = this.$target.find('.modal');
-
-        let display = $main.data('display');
-        let delay = $main.data('showAfter');
 
         if (config.device.isMobile) {
-            if (display === 'onExit') {
-                display = 'afterDelay';
-                delay = 5000;
+            if (this.display === 'mouseExit') {
```

Comment:
- We cannot rename that... what about compatibility? EDIT: I see 'mouseExit' was introduced this way, so it seems right... but does not that mean this is a bug fix for stable versions?
- Also: can't this be moved to the start method as well? Feels strange to have moved the definition of this.display there but not this part

Reply:
- Yes it's actually a bug in stable, I add it in the "test website" task right now